README.md

```diff
@@ -59,33 +59,24 @@ Setting `noReset` to `true` will leave the simulator running at the end of a tes
 
 The `appium-xcuitest-driver` has provisional support for iOS real devices. Not all functionality is currently supported.
 
-The main issue with testing on real devices is that the app under test must be built with the same provisioning profile as the WebDriverAgent is built. At the moment, the command being used to build the WebDriver agent is (with adjustments for paths):
+WebDriverAgent needs to be built with development team and provisioning profile installed on device.
 
-```
-xcodebuild \
-  -project' <path-to-WebDriverAgent>/WebDriverAgent.xcodeproj \
-  -scheme WebDriverAgentRunner \
-  -sdk iphoneos \
-  -destination platform=iOS,id=<UDID> \
-  CODE_SIGN_IDENTITY=iPhone Developer \
-  CODE_SIGNING_REQUIRED=YES \
-  test
-```
+* Open terminal go to `/appium-xcuitest-driver/WebDriverAgent` and then run `sh ./Script/build.sh`
```

Comment:
This path will be confusing to most people, I think. If they've installed globally then where is it?